src/overlays/actors/ovl_En_Ru2/z_en_ru2.c

```diff
@@ -782,34 +986,34 @@ void EnRu2_Init(Actor* thisx, PlayState* play) {
     EnRu2* this = (EnRu2*)thisx;
 
     ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawCircle, 30.0f);
-    func_80AF2550(thisx, play);
+    EnRu2_InitCollider(thisx, play);
     SkelAnime_InitFlex(play, &this->skelAnime, &gAdultRutoSkel, NULL, this->jointTable, this->morphTable, 23);
 
-    switch (func_80AF26A0(this)) {
+    switch (EnRu2_GetType(this)) {
         case 2:
-            func_80AF2E1C(this, play);
+            EnRu2_InitCrossingArms(this, play);
```

Comment:
What is crossing arms? Like what cutscne does this correspond to? Might be helpful to name it after that, so the corresponding type name is also clear.

Reply:
Two places. One is when you complete the Water Trial, the other is during the "Seal Ganon" cutscene.
It's named after the pose she strikes where she crosses her arms in an X.